nqeit-cross-engine/build.gradle.kts

```diff
@@ -35,7 +35,11 @@ dependencies {
   icebergSparkDependencies("implementation", crossEngine.sparkScala)
   icebergFlinkDependencies("implementation", crossEngine.flink)
 
-  commonTestDependencies()
+  testImplementation(platform(libs.junit.bom))
```

Comment:
Why is it `junit.bom` but not `junit-bom` (as in the TOML)?

Reply:
Kotlin identifier's don't like `-`